From fbc87969a23cb5a8f05d15337c587db9b79089ba Mon Sep 17 00:00:00 2001
From: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Date: Mon, 8 May 2023 14:50:11 -0300
Subject: [PATCH] csm-observability: change powerstore secret name

This patch consists of changing the secretName used from csi-powerstore.
It's needed because the secret of the powerstore is created based on
the default release name and it was used hard-coded here.

Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
---
 .../templates/karavi-metrics-powerstore.yaml                    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
index a54d1d5..8fcd046 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
@@ -76,7 +76,7 @@ spec:
               path: localhost.key
       - name: powerstore-config
         secret:
-          secretName: powerstore-config
+          secretName: csi-powerstore-config
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
-- 
2.25.1
